src/pages/iou/request/step/IOURequestStepAmount.tsx

```diff
@@ -184,10 +184,10 @@ export default withWritableReportOrNotFound(
                     return `${ONYXKEYS.COLLECTION.SPLIT_TRANSACTION_DRAFT}${transactionID}`;
                 },
             },
-            backupTransaction: {
+            draftTransaction: {
```

Comment:
This is how staging looks like https://github.com/Expensify/App/blob/staging/src/pages/iou/request/step/IOURequestStepAmount.tsx